admin/meetings/111624-sprint-1-review.md

```diff
@@ -0,0 +1,76 @@
+# Sprint Review Meeting - Sprint 1
+
+**Date:** Nov 16, 2024 <br>
+**Location:** Virtual Meeting
+
+## Welcome and Introduction
+
+- Meeting purpose: Review of Sprint 1 and gather feedback
+- Attendees: Tanmay, Mokshita, Kshitij, Angus, Chris, Fude, Jyotika, Rohan, Chris C
+
+## Sprint Goal Review
+
+Sprint goal: 
+- Understand the existing VSCode extension code and identify the complexities involved in implementing the proposed features. 
+- Analyze the tools to use for linting, code styling, code quality, documentation, unit and/or e2e testing.
+<br>
+
+Goal achievement status: Partially achieved - need to try out code quality tool integration
+
+## Sprint Backlog Review 
+
+### Completed items:
+
+![alt text](111624-sprint-1-done.png)
```

Comment:
Type/copy-paste these in. Images aren't indexable/searchable which limits the utility of this document.

Reply:
Noted.